docs/up-to-date-check.md

```diff
@@ -1,17 +1,46 @@
 ﻿# Up-to-date Check
 
-The Project System's _Fast Up-to-date Check_ saves developers time by quickly assessing whether a project needs to be
+The Project System's _Fast Up-to-date Check_ (FUTDC) saves developers time by quickly assessing whether a project needs to be
```

Comment:
```suggestion
The Project System's _Fast Up-to-Date Check_ (FUTDC) saves developers time by quickly assessing whether a project needs to be
```

I would call this a nit but I'm not sure it rises to that level. 🤣 I would tend to write this as "Fast Up-to-Date Check" as it is a proper name; that leads me to believe all the significant words should be capitalized. It is just as likely, though, that my understanding of grammar and punctuation is lacking. :-)

Reply:
Interesting. I think you're right. I wonder if FUTDC should actually be FUDC too!